File: app/src/main/java/com/ryanrvldo/moviecatalogue/data/source/model/Similar.java
package com.ryanrvldo.moviecatalogue.data.source.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import lombok.Data;

@Data
public class Similar {
    @Expose
    private int id;

    @Expose
    private String title;

    @Expose
    private String name;

    @SerializedName("poster_path")
    @Expose
    private String posterPath;

    @SerializedName("vote_average")
    @Expose
    private float rating;
}

File: app/src/main/kotlin/com/ryanrvldo/moviecatalogue/data/model/Similar.kt
package com.ryanrvldo.moviecatalogue.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Similar(
    @Expose val id: Int,
    @Expose val title: String,
    @Expose val name: String,
    @Expose @SerializedName("poster_path") val posterPath: String,
    @Expose @SerializedName("vote_average") val rating: Float
)